erts/emulator/beam/jit/beam_asm.cpp

```diff
@@ -147,7 +149,16 @@ void beamasm_init() {
 
     cpuinfo = CpuInfo::host();
 
-    bga = new BeamGlobalAssembler();
+    JitAllocator::CreateParams params;
+
+    params.options = JitAllocator::kOptionUseDualMapping |
+                     JitAllocator::kOptionUseMultiplePools |
+                     JitAllocator::kOptionFillUnusedMemory;
+    params.blockSize = 4 << 20;
```

Comment:
I understand why you use DualMapping, but maybe add a comment about the other options and the blockSize?